common/djangoapps/student/tests/test_views.py

```diff
@@ -157,91 +154,6 @@ def test_course_run_refund_status_invalid_course_key(self):
         self.assertEqual(response.status_code, 406)
 
 
-@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-class LogoutTests(TestCase):
```

Comment:
@nasthagiri: I think we missed moving these tests when we moved the view in the refactor PR.  Moving this now since I'll need to add a test.